budgetportal/models.py

```diff
@@ -9,6 +9,7 @@
 from itertools import groupby
 from pprint import pformat
 from urllib.parse import urljoin, quote
+from uuid import uuid4
```

Comment:
we don't need this any more, do we?

Reply:
True, to remove.

Reply:
Not it cannot be  `irm_snapshot_file_path` uses it

Reply:
why is it showing as a new line in this diff then? :)

Reply:
irm_snapshot_file_path uses the `uuid` import higher up - unless that's changed, in which case you can delete the bare uuid import